File: Modul-1/C/PRAK104-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
    int hargaSepatuA;
    printf("Harga Sepatu A adalah ");
    scanf("%d", &hargaSepatuA);
    int hargaSepatuB;
    printf("Harga Sepatu B adalah ");
    scanf("%d", &hargaSepatuB);
    float diskonSepatuA = 0.13;
    float diskonSepatuB = 0.21;
    int hargaSepatuADiskon = hargaSepatuA * (1 - diskonSepatuA);
    int hargaSepatuBDiskon = hargaSepatuB * (1 - diskonSepatuB);

    printf("Harga sepatu A setelah mendapat diskon menjadi: %d\n", hargaSepatuADiskon);
    printf("Harga sepatu B setelah mendapat diskon menjadi: %d\n", hargaSepatuBDiskon);

    return 0;
}

File: Modul-1/python/PRAK104-2310817110006-Adrian_Bintang_Saputera.py
harga_sepatu_A = int(input("Harga Sepatu A adalah "))
harga_sepatu_B = int(input("Harga Sepatu B adalah "))

diskon_sepatu_A = 0.13
diskon_sepatu_B = 0.21

harga_sepatu_A_diskon = int(harga_sepatu_A * (1 - diskon_sepatu_A))
harga_sepatu_B_diskon = int(harga_sepatu_B * (1 - diskon_sepatu_B))

print(f"Harga sepatu A setelah mendapat diskon menjadi: {harga_sepatu_A_diskon}")
print(f"Harga sepatu B setelah mendapat diskon menjadi: {harga_sepatu_B_diskon}")
